app/src/main/java/com/github/partymakers/partymaker/activities/LoginActivity.java

```diff
@@ -0,0 +1,22 @@
+package com.github.partymakers.partymaker.activities;
+
+import androidx.appcompat.app.AppCompatActivity;
+import com.github.partymakers.partymaker.R;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+
+public class LoginActivity extends AppCompatActivity {
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+    }
+
+    public void loginSuccessful(View view) {
+        Intent intent = new Intent(this, MainActivity.class);
+        startActivity(intent);
```

Comment:
Use `finish` method instead.

Reply:
Also, if it's possible, make `loginSuccessfull()` private.

Reply:
I'm afraid it's not possible. The app crashes in both cases. 
<img width="275" alt="image" src="https://user-images.githubusercontent.com/61482996/112703776-5d836f00-8e98-11eb-8003-0b9e41efbdc1.png">

Reply:
leave it public then